components/custom-network/repositories-list.tsx

```diff
@@ -40,6 +41,10 @@ export default function RepositoriesList({ withLabel = true, repositories, onCli
 
     setReposWithIssues(repositories.filter(repository => repository.hasIssues)
       .map((repository) => repository.fullName));
+
+    setReposUserNotAdmin(repositories.filter(repository => repository.userPermission !== "ADMIN")
```

Comment:
shouldn't "mantainer" be enough ? [honest question, I could not find the docs for gh roles]